src/python/grpcio_tests/tests/protoc_plugin/_python_plugin_test.py

```diff
@@ -504,7 +504,8 @@ def half_duplex_request_iterator():
         service.server.stop(None)
 
 
-@unittest.skipIf(sys.version_info[0] < 3, "Unsupported on Python 2.")
+@unittest.skipIf(sys.version_info[0] < 3 or sys.version_info[1] < 6,
```

Comment:
optional: This expression feels weird. But Python is unlikely to have 2.8 ever, or 4.0 soon.